kola/tests/ignition/resource.go

```diff
@@ -128,6 +128,22 @@ func init() {
 		  }
 	      }`),
 	})
+	register.Register(&register.Test{
+		Name:        "coreos.ignition.v2_1.resource.s3_config",
+		Run:         remoteS3Config,
+		ClusterSize: 1,
+		Platforms:   []string{"aws"},
+		UserData: conf.Ignition(`{
+			"ignition": {
+				"version": "2.1.0",
+				"config": {
+					"append": [{
+						"source": "s3://kola-fixtures/resources/config.ign"
```

Comment:
Let's rename the object to `authenticated.ign` to clarify that the ACL is relevant.